addons/damage/stringtable.xml

```diff
@@ -580,59 +580,70 @@
             <Portuguese>ERA</Portuguese>
             <Russian>ДЗ</Russian>
             <Japanese>ERA</Japanese>
-            <Korean>폭발반응장갑</Korean>
+            <Korean>폭발반응장갑차</Korean>
             <Chinesesimp>爆炸式反应装甲</Chinesesimp>
             <Chinese>爆炸式反應裝甲</Chinese>
         </Key>
         <Key ID="STR_ZEN_Damage_Hitera_front">
             <English>ERA Front</English>
             <German>Reaktivpanzerung vorne</German>
             <Polish>ERA Przód</Polish>
+            <Korean>폭발반응장갑차 앞</Korean>
+            <Polish>ERA Przód</Polish>
```

Comment:
Duplicate translation. Remove this line.

Reply:
Fixed it 👍